mito-ai/mito_ai/streamlit_conversion/streamlit_agent_handler.py

```diff
@@ -32,31 +43,39 @@ async def generate_new_streamlit_code(notebook: List[dict], streamlit_app_prompt
     ]
     agent_response = await get_response_from_agent(messages)
     converted_code = extract_code_blocks(agent_response)
-    
-    # Extract the TODOs from the agent's response
-    todo_placeholders = extract_todo_placeholders(agent_response)
-    
-    for todo_placeholder in todo_placeholders:
-        print(f"Processing AI TODO: {todo_placeholder}")
-        todo_prompt = get_finish_todo_prompt(notebook, converted_code, todo_placeholder)
```

Comment:
Now that we are going cell by cell I think it might be reasonable to not have the todo's anymore, but to delete it, I think we need to:

1. Let's delete the `get_finish_todo_prompt` and `extract_todo_placeholders` functions since we're not using them anymore 
2. Update the `get_streamlit_app_creation_prompt` function so it no longer tells the model to use TODOs

Then, let's make sure to test this [with this notebook](https://github.com/mito-ds/mito/blob/dev/mito-ai/example-notebooks/geospatial/bike-violations.ipynb) (its in the mito-ai/example-notebooks/geospatial folder) and confirm that it still moves over the entire `violation_data_list`